pulse_actions/handlers/talos.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module is for the following use case:
+
+ - Talos jobs:
+
+    * Trigger talos jobs twice if they are from PGO build.
+"""
+import logging
+
+from mozci import query_jobs
+from pulse_actions.utils.misc import filter_invalid_builders
```

Comment:
Could you please sort the modules?
Also move the pulse_actions module under its own section (one blank line in between).
We like to separate the block of external modules visually.

This is referenced in here:
http://ateam-bootcamp.readthedocs.io/en/latest/reference/python-style.html#import-statements